[FIX] microsoft_outlook: redirect URL error if not matching (#10303)
odoo/documentation: content/applications/general/email_communication/azure_oauth.rst

and personal Microsoft accounts (e.g. Skype, Xbox)`.

Under the :guilabel:`Redirect URL` section, select :guilabel:`Web` as the platform, and then input
`https://<web base url>/microsoft_outlook/confirm` in the :guilabel:`URL` field. The `web.base.url`
is subject to change depending on the URL used to log-in to the database.

.. note::

>    Our documentation about the :ref:`web.base.url <domain-name/web-base-url>` explains how to freeze

Reviewer: ```suggestion
   The documentation about the :ref:`web.base.url <domain-name/web-base-url>` explains how to freeze
```

>    a unique url. It's also possible to add different Redirect URLs on the Microsoft app.

Reviewer: ```suggestion
   a unique url. It is also possible to add different redirect URLs on the Microsoft app.
```

> After the URL has been added to the field, :guilabel:`Register` the application so it is created.

Reviewer: ```suggestion
After the URL has been added to the field, :guilabel:`Register` the application, so it is created.
```

and personal Microsoft accounts (e.g. Skype, Xbox)`.

Under the :guilabel:`Redirect URL` section, select :guilabel:`Web` as the platform, and then input
`https://<web base url>/microsoft_outlook/confirm` in the :guilabel:`URL` field. The `web.base.url`

> is subject to change depending on the URL used to log-in to the database.

Reviewer 2: ```suggestion
is subject to change depending on the URL used to log in to the database.
```

.. note::
   The documentation about the :ref:`web.base.url <domain-name/web-base-url>` explains how to freeze

>    a unique url. It is also possible to add different redirect URLs on the Microsoft app.

Reviewer 2: ```suggestion
   a unique URL. It is also possible to add different redirect URLs on the Microsoft app.
```
